common/src/model.ts

```diff
@@ -47,6 +47,7 @@ export interface CardModel extends CardTextFieldValues {
     readonly audio?: AudioModel;
     readonly file?: FileModel;
     readonly mediaTimestamp: number;
+    readonly exportMode?: AnkiExportMode;
```

Comment:
I understand this might be convenient for passing `exportMode` around with the card, but would it be possible to instead add `exportMode` directly to the message interfaces instead?